test/fetch-adapter/fetch-adapter.spec.ts

```diff
@@ -19,6 +20,7 @@ describe('fetch adapter', () => {
       {
         [handler.method]: async (ctx: any) => {
           receivedContext = ctx;
+          await setTimeout(100);
```

Comment:
would 0245294db4de43f7b2baf8279d7a4abe21d780e4 avoid issues with using timers?

Reply:
That looks like the earlier version I had 🤔 ... Could we just get this PR merged with that commit or not, I don't like going back and forth and wasting everyones time. @sakari

Reply:
you mean [this](https://github.com/smartlyio/oats/pull/444/commits/737db06501e65828966d448c11e11d76a2c0641b). As far as I can see with my commit we would not rely on timings whereas with that commit we would rely on the server not resolving the request before the abort controller fires.

Reply:
fine with that. Could we avoid force pushing stuff while things are in flight because it makes collaboration way less fun.

Reply:
Yeah, bad habbit